client/src/ui/molecules/login/index.scss

```diff
@@ -4,8 +4,14 @@
 .auth-container {
   margin-top: $base-unit * 2;
 
-  @media #{$mq-large-desktop-and-up} {
-    margin-top: 0;
+  /* This media condition could be removed as margin-top isn't changing from above */
+  // @media #{$mq-large-desktop-and-up} {
+  //   margin-top: 0;
+  // }
+
+  /* Handling < 1199px Sign in button alignment, if there is a $var then use that instead */
+  @media (max-width: 74.9375em) and (min-width: 768px) {
+    margin-top: $base-unit * 3;
   }
```

Comment:
We should not introduce new media queries. We have defined media queries to use for this purpose. We should always follow a mobile-first approach.

These are the media queries to use:

```
$mq-large-desktop-and-up: "all and (min-width: #{$small-desktop-ends})";
$mq-small-desktop-and-up: "all and (min-width: #{$small-desktop-starts})";
$mq-tablet-and-up: "all and (min-width: #{$tablet-starts})";
```

For example:

```
@media #{$mq-tablet-and-up} {
   ...
}
```

Reply:
@schalkneethling Got it! I will make sure to update this.